docs/changelog/109435.yaml

```diff
@@ -1,5 +1,5 @@
 pr: 109435
-summary: Adding `ActionRequest-based` reranker
+summary: Adding support for InferenceAPI-based reranking 
```

Comment:
I don't understand, this doesn't add the inference action stuff does it? I thought this was the base logic that will be used later?

Reply:
yeap;  this is just the base class hierarchy for adding the inference ranking logic later.. Apologies for not making this clear enough in the changelog :/ (updated it because the abstract class `ActionRequest*` has been removed so this seemed a bit outdated); would rephrasing to  
```
adding Reranking*Context classes to support global reranking
```
make more sense?

Reply:
@pmpailis I think so. But I am not sure this is a feature nor enhancement? There are no user facing improvements. So, maybe we don't need release notes at all and we switch to `>non-issue` ?

Reply:
yeah, that makes sense! will convert :)